src/main/java/seedu/address/model/time/HourBlock.java

```diff
@@ -84,6 +84,13 @@ public void setCommitment(Commitment commitment) {
         }
     }
 
+    /**
+     * Removes the commitment from the HourBlock.
+     */
+    public void removeCommitment() {
+        commitment = Optional.empty();
```

Comment:
Thanks for the addition for completeness!

Reply:
Thanks!